docs/advanced-usage.md

```diff
@@ -19,6 +19,21 @@ steps:
 - run: npm test
 ```
 
+## Node version file
+
+The `node-version-file` input allows you to use a file within your repository which contains the version of node your project uses for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used.
```

Comment:
I think it makes sense to add a link to the [Supported version syntax](https://github.com/actions/setup-node#supported-version-syntax) section

Reply:
@MaksimZhukov im wondering if we should remove  `If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used.` since it will throw a warning to the user for using both inputs